packages/maskbook/src/components/InjectedComponents/PostDialog.tsx

```diff
@@ -217,7 +218,7 @@ export function PostDialogUI(props: PostDialogUIProps) {
                             }}>
                             <ClickableChip
                                 checked={props.imagePayload}
-                                label={t('post_dialog__image_payload')}
+                                label={t('post_dialog__image_payload') + Flags.has_no_browser_tab_ui ? `(Beta)` : ''}
```

Comment:
This will also add `beta` to Android App. Your commit writes `add (Beta) to image payload option on iOS`. Is that expected?

Reply:
Thought Android GeckoView has browser tab api.
Should I use `support_native_injected_script_declaration ` instead?

Reply:
No. What exact description do you want? On iOS or all Apps (iOS and Android)?

Reply:
Only on iOS, but it seems we do not export the const `is_iOSApp ` from our `flags.ts`

Reply:
https://github.com/DimensionDev/Maskbook/blob/e46948584980c7b3380cd53806496a0eda9233b3/packages/maskbook/src/components/InjectedComponents/PostDialog.tsx#L134-L142

Reply:
@septs this is not a plugin

Reply:
@jk234ert so why it's beta on iOS but not on android?

Reply:
Confirmed with design, should add the `(Beta)`  sub on both mobile platforms

Reply:
Hold on let me change it